main/test/ca/uwaterloo/flix/library/TestArray.flix

```diff
@@ -3957,155 +3957,323 @@ def formatWith06(): Bool & Impure =
     s == "1..2..3"
 
 /////////////////////////////////////////////////////////////////////////////
-// sortWith                                                                  //
+// sortWith                                                                //
 /////////////////////////////////////////////////////////////////////////////
 
+def cmp(x: Int32, y: Int32): Comparison =
+    if (x < y) LessThan
+    else if (x == y) EqualTo
+    else GreaterThan
+
 @test
 def sortWith01(): Bool & Impure =
-    let a = Array.sortWith((x,y) -> if (x < y) -1 else 1, []: Array[Int32]);
-    Array.sameElements(a, []: Array[Int32])
```

Comment:
Should we remove `Array.sameElements` to avoid  confusion? 

(Do we have other functions called `sameElements`?)

Reply:
Yes this should be changed; there is already #1960 where I just added a comment. I suggest we take it from there, independently.